themes-default/slim/static/js/ajax-episode-search.js

```diff
@@ -55,7 +55,9 @@ function updateImages(data) {
                 // Update Status and Quality
                 let qualityPill = '';
                 if (ep.quality_style !== 'na') {
-                    qualityPill = ' <span class="quality ' + ep.quality_style + '">' + ep.quality_name + '</span>';
+                    // @FIXME: (sharkykh) This is a hack to get the scoped style to work.
+                    const qualityPillScopeId = window.components.find(c => c.name === 'quality-pill')._scopeId;
+                    qualityPill = ' <span ' + qualityPillScopeId + ' class="quality ' + ep.quality_style + '">' + ep.quality_name + '</span>';
```

Comment:
It's a hack, but it can be removed once this file gets implemented in Vue.
I think implementing an `EpisodeRow` component would be best suited for this, BTW.

Reply:
I could see the `EpisodeRow` component being used a few places.